pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -516,6 +516,10 @@ multizone:
       # Response backoff is a time Global CP waits before sending ACK/NACK.
       # This is a way to slow down Zone CP from sending resources too often.
       responseBackoff: 0s # ENV: KUMA_MULTIZONE_GLOBAL_KDS_RESPONSE_BACKOFF
+      tracing:
```

Comment:
shouldn't this be somewhere under just `tracing`?

Reply:
IMO no, this concerns only KDS. Where as tracing is global